go/cmd/pathwar/api.go

```diff
@@ -157,10 +160,21 @@ func apiCommand() *ffcli.Command {
 }
 
 func svcFromFlags(logger *zap.Logger) (pwapi.Service, *gorm.DB, func(), error) {
+	// logger
+	zapGormLogger := zapgorm2.New(zap.L())
```

Comment:
```suggestion
	zapGormLogger := zapgorm2.New(logger)
```

we don't use globals in pathwar